### PR TITLE
feat: Mark checked in ticket as 'Checked In' on my-ticket page

### LINKED ISSUE #3027: Mark Ticket as "Checked In" on my ticket page ones it is checked in
There is no way to see if a ticket is checked in or not. Please show tickets as status "Checked In" on the my ticket page: https://eventyay.com/my-tickets/

Screenshot from 2019-05-28 11-48-31

### PR DESCRIPTION
**Pull request by shreyanshdwivedi**
Fixes #3027 

#### Short description of what this resolves:
There is no way to see if a ticket is checked in or not. 

#### Changes proposed in this pull request:
 - Shows tickets as status "Checked In" on the my-ticket page

Screenshot_2019-06-04 Completed Upcoming My Tickets Open Event

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by uds5501** [comment]
@shreyanshdwivedi I am facing a weird error

```js
isUserCheckedIn: computed('order.attendees', function() {
    this.order.attendees.forEach( (item) => {
      if (item.isCheckedIn && item.email == this.authManager.currentUser) {
        return 'hello';
      }
      return 'not hello';
    });
    return false;
  })
```
This is the modified isUserCheckedIn which is I am using. Just for testing. I can extract positively that the if condition is working fine but not returning a 'hello' or a 'not hello', instead it throws a false. is there anything like an `await if`?

**Comment by shreyanshdwivedi** [comment]
@uds5501 in my case, I put console.log at appropriate points and found that the true is being returned but false is also being returned for if the user is current_user and is checked in. I don't understand that once true the following code should not be executed but it is being executed.
Maybe @CosmicCoder96 can tell us if `await if` or any other workaround can work here

**Comment by shreyanshdwivedi** [comment]
@uds5501 I've managed a workaround for this and it works perfectly fine. I just have to fix the test now :)

File: app/templates/components/order-card.hbs [hunk 1]
```diff
@@ -12,6 +12,11 @@
         <div class="ui {{order-color order.status}} large label">
           {{order.status}}
         </div>
+        {{#if isUserCheckedIn}}
+          <div class="ui green large label">
+            {{t 'Checked In'}}
+          </div>
+        {{/if}}
```
Comment (uds5501) [inline app/templates/components/order-card.hbs]: @shreyanshdwivedi an alternate "not checked in" label would be lovely too!

Comment (shreyanshdwivedi) [reply app/templates/components/order-card.hbs]: @uds5501 I've added `Not Checked In` also. Please refer to screenshot attached to PR description
Will work on fixing Travis failure now

**Comment by shreyanshdwivedi** [comment]
@kushthedude the location is perfect IMO. If I place the label to bottom right, it'll look odd.

**Comment by shreyanshdwivedi** [comment]
@CosmicCoder96 @uds5501 @kushthedude please review

**Pull request merged**